js/main.js

```diff
@@ -38,5 +38,112 @@ $codeJournal.addEventListener('submit', function (event) {
 
   $url.src = 'images/placeholder-image-square.jpg';
 
+  $journalList.prepend(renderEntry(journalData));
+
+  viewSwap('entries');
+
+  if (data.entries.length >= 1) {
+    toggleNoEntries();
+  }
+
   $codeJournal.reset();
 });
+
+/*
+ *  <ul class="container">
+ *    <li class="row">
+ *      <div class="column-full column-half">
+ *        <div class="img">
+ *          <img src="" alt="" class="photo">
+ *        </div>
+ *      </div>
+ *      <div class="column-full column-half">
+ *        <h2 class="title"></h2>
+ *        <p class="notes"></p>
+ *      </div>
+ *    </li>
+*/
+
+function renderEntry(entry) {
+  const $firstEntry = document.createElement('li');
+  $firstEntry.setAttribute('class', 'row');
+
+  const $firstDiv = document.createElement('div');
+  $firstDiv.setAttribute('class', 'column-full column-half');
+  $firstEntry.appendChild($firstDiv);
+
+  const $divImg = document.createElement('div');
+  $divImg.setAttribute('class', 'img');
+  $firstDiv.appendChild($divImg);
+
+  const $secondCol = document.createElement('div');
+  $secondCol.setAttribute('class', 'column-full column-half');
+  $firstEntry.appendChild($secondCol);
+
+  const $img = document.createElement('img');
+  $img.setAttribute('src', entry.userPhoto);
+  $img.setAttribute('alt', entry.userTitle);
+  $img.setAttribute('class', 'photo');
+  $divImg.appendChild($img);
+
+  const $h2 = document.createElement('h2');
+  $h2.setAttribute('class', 'title');
+  $h2.textContent = entry.userTitle;
+  $secondCol.appendChild($h2);
+
+  const $p = document.createElement('p');
+  $p.setAttribute('class', 'notes');
+  $p.textContent = entry.userNotes;
+  $secondCol.append($p);
+
+  return $firstEntry;
+}
+
+const $journalList = document.querySelector('.journal-list');
+
+document.addEventListener('DOMContentLoaded', function () {
+  const lastView = data.view;
+  const dataEntries = data.entries;
+  if (lastView) {
+    viewSwap(lastView);
```

Comment:
This `if` statement isn't needed. `data.view` will always have some sort of string, so we should just be calling `viewSwap(lastView)` directly